### DISCUSSION TITLE
How to check if tags have been changed?

### DISCUSSION THREAD

**Discussion by digitall-it** [spatie/laravel-tags#433] [Q&A]
I am using this code to check if tags have changed on a model but it isn't working.

```
static::updating(function($hotel) {
    // if the tags are changed, we need to recalculate the ranking
    if ($hotel->relationLoaded('tags') && $hotel->tags->isDirty()) {
        CalculateHotelRank::dispatch($hotel);
    }
});
```

I get instead the error: `Method Illuminate\Database\Eloquent\Collection::isDirty does not exist.`

**Comment by scramatte** [comment]
Hi,

Have you find a way to achieve this?

Regards
